eventhubs-streamanalytics-eventhubs/tools/eh-asa-perfmon/Program.cs

```diff
@@ -167,24 +32,19 @@ static async Task Main(string[] args)
             {
                 Console.WriteLine("No parameters passed or environment variables set.");
                 Console.WriteLine("Please use --help to learn how to use the application.");
-                Console.WriteLine("  dotnet run -- --help");
+                Console.WriteLine("dotnet run -- --help");
                 return;
             }
 
-            var cts = new CancellationTokenSource();
-            var receiver = new EventReceiver(options.EventHubConnectionString, cts.Token);
-            
-            Task t = receiver.Receive();
-
-            Console.ReadKey(true);
-            cts.Cancel();
+            // As a preventative measure, we'll specify that cancellation should occur after 60 seconds, 
+            // so that we don't iterate indefinitely in the event.
```

Comment:
This sentence is not clear - in the event of what?

Reply:
Resolved, "in the event receiving".